scripts/train_agents.py

```diff
@@ -166,15 +168,41 @@ def set_input(args, quick_test=False):
     sp_w_sp_force_training = True
     
     set_input(args=args, quick_test=quick_test)
+    '''
+    Suggested 3-Chefs Layouts are '3_chefs_small_kitchen_two_resources', 
+    '3_chefs_counter_circuit', '3_chefs_asymmetric_advantages', 
+    '3_chefs_forced_coordination_3OP2S1D'.
+    '''
+    args.n_envs = 50
+    args.learner_type = LearnerType.HELPER
+
+    args.h_dim = 256
+    args.seed = 68
+    SP(args=args,
+       pop_force_training=pop_force_training)
+    
+    args.seed = 13
+    SP(args=args,
+       pop_force_training=pop_force_training)
+    
+    args.seed = 48
+    SP(args=args,
+       pop_force_training=pop_force_training)
+    
+    args.seed = 27
+    SP(args=args,
+       pop_force_training=pop_force_training)
+    
+    args.seed = 6
+    SP(args=args,
+       pop_force_training=pop_force_training)
```

Comment:
do we need to have all these calls to `SP`? 
If not, I think it'd be better to only have one call to `SP` in this file.